### DISCUSSION TITLE
Update schlug fehl

### DISCUSSION THREAD

**Discussion by ChaosChris1975** [tbnobody/OpenDTU#293] [Q&A]
habe aus "OpenDTU Build #322" die "opendtu-generic.bin" auf der Oberfläche ausgewählt und dann erschien auch OTA sucessfull oder so ähnlich. Aber dann ging nichts mehr.

Hab dann versucht über das Flash-Download-Tool von Espressif die alte Version die ich vorher drauf hatte zu flashen, das funktioniert aber auch nicht.

Hier der serielle LOG nach reset:
21:58:13.022 -> ets Jul 29 2019 12:21:46
21:58:13.022 -> 
21:58:13.022 -> rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
21:58:13.022 -> configsip: 0, SPIWP:0xee
21:58:13.022 -> clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
21:58:13.022 -> mode:DIO, clock div:2
21:58:13.022 -> load:0x3fff0030,len:1184
21:58:13.022 -> load:0x40078000,len:12784
21:58:13.022 -> load:0x40080400,len:3032
21:58:13.062 -> entry 0x400805e4
21:58:13.622 -> 
21:58:13.622 -> Starting OpenDTU
21:58:13.622 -> Initialize FS... done
21:58:13.622 -> Reading configuration... done
21:58:13.662 -> Initialize Network... done
21:58:13.782 -> Setting Hostname... Initialize NTP... done
21:58:13.782 -> Initialize MqTT... done
21:58:13.782 -> Initialize WebApi... done
21:58:13.822 -> Initialize Hoymiles interface... Connection successfull
21:58:13.822 ->   Setting radio PA level... 
21:58:13.822 ->   Setting DTU serial... 
21:58:13.822 ->   Setting poll interval... 
21:58:13.822 -> done
21:58:13.822 -> Switch to WiFi mode
21:58:13.822 -> Setting Hostname... done
21:58:29.622 -> Disable search for AP... done

Jemand eine Idee ??

**Comment by tbnobody** [comment]
Wenn du eine sehr alte Version drauf hattest und auf eine neue Upgedated hast wurde die Config in ein neues Format konvertiert. Das neue Format kann die alte Version aber nicht mehr lesen. Dadurch sieht es jetzt so aus als hättest du gar keine Config mehr eingegeben. Flashe einfach die neue Version mit dem ESP Flash Tool.

>  Aber dann ging nichts mehr.

Was war denn zu dem Zeitpunkt die Ausgabe des seriellen Logs?

**Reply by ChaosChris1975** [reply]
Danke, hab nochmals die kompletten 4 Files geflasht, jetzt läuft es wieder.
Hoffe er findet morgen früh auch den Wechselrichter.

War eine Version von mitte Oktober, also nicht so alt.
 
Serielle Log ist oben, dann kam nichts mehr serielles danach, und auch kein Zugriff auf den ESP.
